addons/website_slides/models/slide_channel.py

```diff
@@ -477,6 +471,12 @@ def _compute_website_background_image_url(self):
                      % ('training' if channel.channel_type == 'training' else 'documentation')
             )
 
+    def _get_placeholder_filename(self, field):
+        image_fields = ['image_%s' % size for size in [1920, 1024, 512, 256, 128]]
+        if field in image_fields:
+            return self._compute_website_background_image_url()
```

Comment:
You don't need to call the method (which doens't return anything), but the field directly.